docs/supported-libraries.md

```diff
@@ -225,6 +225,9 @@ For this reason, the following instrumentations are disabled by default:
 - `jdbc-datasource` which creates spans whenever the `java.sql.DataSource#getConnection` method is called.
 - `dropwizard-metrics` which might create a very low quality metrics data, because of lack of label/attribute support
   in the Dropwizard metrics API.
+- `mybatis`
+- `spring-boot-actuator-autoconfigure`
+- `spring-batch`
```

Comment:
It seems there are other disabled instrumentations that you ignored. Such as micrometer.https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/70e4d0522324b830f2ff9019ab231b2d9fbdeb14/instrumentation/micrometer/micrometer-1.5/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/micrometer/v1_5/MicrometerInstrumentationModule.java#L26

You can find all of them by searching `public boolean defaultEnabled(ConfigProperties config) {` and checking which one's default value is false.

Reply:
Not sure whether I should include the internal instrumentations, but I've added all the others in 0c2b870f1dbf3ee94e8fcb1c00a824019b328cbd